.bazelrc

```diff
@@ -1,23 +1,14 @@
-common --noenable_bzlmod
-common --enable_workspace
-# Avoid being broken by version requirement changes in transitive deps.
-common --check_direct_dependencies=error
-
-common --enable_platform_specific_config
+# Import shared config
+try-import %workspace%/shared.bazelrc
```

Comment:
I think we probably want to include a big notice at the top of the file that folks should prefer to edit the shared config unless they are specifically making a change that should only apply to the OSS repo. It's rare that we want OSS-only bazelrc configs, since the internal repo is mostly just the OSS repo (as an external dep) plus some configs / k8s targets.

Reply:
Added a small noticed at the top of the file.

I do hope the file is short-lived though.